### DISCUSSION TITLE
Combobox max-height for options dropdown

### DISCUSSION THREAD

**Discussion by sbruinsje** [tailwindlabs/headlessui#3288] [Help]
**Version**: headlessui 2.0.3.
**Component**: [Combobox](https://headlessui.com/react/combobox)

I am trying to control the max height of the ComboboxOptions. I tried to just setting a maximum height tailwind class but its not working (`<ComboboxOptions className="max-h-[200px]" >`). I noticed the rendered element sets a style property on the element iself: `max-height: min(var(--anchor-max-height, 100vh), 571px)`.

It looks like you're supposed to control the max-height through a css variable. This seems to work but its not documented on the [Combobox react docs](https://headlessui.com/react/combobox). Also it would be nice if you can control it through other means as well like an attribute.

Is there more info on this available?

**Comment by antonsamper** [comment]
Not ideal but until there is a better solution you can always use the dreaded `!important` e.g.

```
<ComboboxOptions className="!max-h-[200px]">
   ...
</ComboboxOptions>
```
